casper/src/main/scala/coop/rchain/casper/util/DoublyLinkedDagOperations.scala

```diff
@@ -64,7 +64,7 @@ object DoublyLinkedDagOperations {
     val parentToChildAdjacencyList: Map[A, Set[A]] = dag.parentToChildAdjacencyList
     val childToParentAdjacencyList: Map[A, Set[A]] = dag.childToParentAdjacencyList
     assert(!childToParentAdjacencyList.contains(element))
-    assert(!parentToChildAdjacencyList.values.toSet.contains(element))
+    assert(!parentToChildAdjacencyList.values.toSet.contains(Set(element)))
```

Comment:
Probably not really worth a change but you could do: `parentToChildAdjacencyList.values.exists(_ == Set(element))`.